### PR TITLE
Change type of column 'info' in table 'query_info' to jsonb

### PR DESCRIPTION
**Pull request by radek-starburst**
The column 'info' in the table 'query_info' has a type 'text'. This column stores only json documents so it is reasonable to change type of this column to jsonb.

### CODE REVIEW

**Review by nineinchnick — APPROVED** [review]

**Review by przemekak — APPROVED** [review]

File: benchto-service/pom.xml [hunk 1]
```diff
@@ -129,6 +129,10 @@
             <artifactId>hibernate-validator</artifactId>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>com.vladmihalcea</groupId>
```
Comment (sopel39) [inline benchto-service/pom.xml:133]: Is there a more official library?

**Review by sopel39 — APPROVED** [review]
lgtm % comment

File: benchto-service/pom.xml (hunk 1 above)
Comment (radek-starburst) [reply benchto-service/pom.xml:133]: I cannot find the official library. Anyway, `hibernate-types-52` commonly used.

Comment (nineinchnick) [reply benchto-service/pom.xml:133]: I follow the author, he's pretty active in the Java community.

**Pull request merged**